deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -3885,6 +3885,12 @@ spec:
                   pause:
                     type: string
                     description: Desired image for pause image.
+                  dynamicPlugin:
+                    type: string
+                    description: Desired image for dynamic plugin image.
+                  nginxinc:
+                    type: string
+                    description: Desired image for nginx-unprivileged image.
```

Comment:
Let's just say `Desired image for nginx proxy`. We don't have to mention what flavor we are using.